PowerPointLabs/PowerPointLabs/PositionsLab/PositionsLabMain.cs

```diff
@@ -2097,6 +2097,49 @@ public static void Swap(List<PPShape> selectedShapes, bool isPreview)
             }
         }
 
+        public static void Repack(List<PPShape> selectedShapes, bool isPreview)
+        {
+            var numOfShapes = selectedShapes.Count;
+
+            if (numOfShapes < 2)
+            {
+                throw new Exception(ErrorMessageFewerThanTwoSelection);
+            }
+
+            var sortedShapes = selectedShapes;
+            System.Drawing.PointF[] topLeftPointList = new System.Drawing.PointF[numOfShapes];
+            sortedShapes = Graphics.SortShapesByLeft(selectedShapes);
+            float[] distanceList = new float[numOfShapes - 1];
+            var currentX = sortedShapes[0].ActualTopLeft.X;
+
+            for (var i = 0; i < numOfShapes - 1; i++)
```

Comment:
Can you merge the first and second `for` loop together by initializing the `topLeftPointList[0]` then loop starting from 1 and use `i` and `i-1` instead? Can you also merge line 2109 and 2111?